src/PublishPort.jsx

```diff
@@ -57,8 +102,12 @@
                             step={1}
                             min={1}
                             max={65535}
+                            validated={validationFailed?.containerPort ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-122305-68cb803a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)